pkg/api/v1/types.go

```diff
@@ -2477,6 +2491,23 @@ type ServiceProxyOptions struct {
 	Path string `json:"path,omitempty" protobuf:"bytes,1,opt,name=path"`
 }
 
+// OwnerReference contains enough information to let you inspect or modify the
+// owner object. Owner could be an object in the same namespace as the
+// dependent, or could be root scoped, so namespace is not a field.
```

Comment:
Last sentence: An owning object must be in the same namespace, so there is no namespace field.

(Current wording makes it sound like you can refer to root scoped objects from anywhere, which is not true.)